### PR TITLE
Move global parameters to a dedicated sidebar css class.

### PR DESCRIPTION
**Pull request by kberg**
This is the second step in reworking the sidebar a bit.

### CODE REVIEW

File: src/components/Sidebar.ts [hunk 1]
```diff
@@ -193,161 +193,167 @@ export const Sidebar = Vue.component('sidebar', {
...
-                      <game-setup-detail :gameOptions="gameOptions" :playerNumber="playerNumber" :lastSoloGeneration="lastSoloGeneration"></game-setup-detail>
+<div :class="'sidebar_cont sidebar '+getSideBarClass()" :data="syncPreferences()">
+  <div class="tm">
+    <div class="gen-text">GEN</div>
+    <div class="gen-marker">{{ getGenMarker() }}</div>
+  </div>
+  <div v-if="gameOptions.turmoilExtension">
+    <div :class="'party-name party-name-indicator party-name--'+rulingPartyToCss()" v-html="getRulingParty()"></div>
+  </div>
+  <div class="global_params">
```
Comment (kberg) [inline src/components/Sidebar.ts:204]: 1. Be sure to hide whitespace changes
2. Note dropping the "preferences" prefix as this is using the sidebar namespace.

File: src/components/Sidebar.ts [hunk 2]
```diff
@@ -193,161 +193,167 @@ export const Sidebar = Vue.component('sidebar', {
...
+          <i class="form-icon"></i>
+          <span v-i18n>Learner Mode (req. refresh)</span>
+          <span class="tooltip tooltip-left" data-tooltip="Show information that can be helpful\n to players who are still learning the games">&#9432;</span>
+        </label>
+      </div>
+      <div class="preferences_panel_item form-group">
+        <label class="form-label"><span v-i18n>Language</span> (<a href="javascript:document.location.reload(true);" v-i18n>refresh page</a> <span v-i18n>to see changes</span>)</label>
+        <div class="preferences_panel_langs">
+          <label class="form-radio" v-for="language in langs">
+            <input name="lang" type="radio" v-on:change="updatePreferences" v-model="lang" :value="language.id"></input>
```
Comment (bafolts) [inline src/components/Sidebar.ts:345]: I don't believe it is permitted for the `input` element to have a closing tag.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input

```
Tag omission | Must have a start tag and must not have an end tag.
```

Comment (kberg) [reply src/components/Sidebar.ts:345]: I removed it from everywhere. What a weird rule.

Comment (bafolts) [reply src/components/Sidebar.ts]: I had to look this up. I vaguely remembered this rule and then took the time to find the proof. Is very strange.

**Review by bafolts — APPROVED** [review]

**Pull request merged**